Makefile

```diff
@@ -75,9 +75,12 @@ build-unit-tests-image:
 run-unit-tests:
 	$(run_unit_tests_image) make test
 
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 .PHONY: test
 test: check-generated-manifests update
-	ginkgo -r -v -skipPackage tests
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
```

Comment:
why `v0.8.3` and not higher (e.g. [v0.9.2](https://github.com/IBM/ibm-block-csi-operator/blob/v1.8.0/go.mod#L17))?
consider using a [newer "infra"](https://github.com/kubernetes-sigs/controller-runtime/tree/v0.9.2/tools/setup-envtest#envtest-binaries-manager)

Reply:
done